### ISSUE TITLE
Type 'ChatAnthropic' is not assignable to type 'LLMType'.

### ISSUE THREAD

**Issue by AdiletBekmuratov** [langchain-ai/langchainjs#4056]
Hello! Today after updating langchain package I encountered such error:

Type 'ChatAnthropic' is not assignable to type 'LLMType'.
  Type 'ChatAnthropic' is not assignable to type 'LLMType'.
    Type 'ChatAnthropicMessages<BaseLanguageModelCallOptions>' is not assignable to type 'Runnable<BaseLanguageModelInput, BaseMessage, RunnableConfig>'.
      Property 'lc_runnable' is protected but type 'Runnable<RunInput, RunOutput, CallOptions>' is not a class derived from 'Runnable<RunInput, RunOutput, CallOptions>'.ts(2322)
      
 Example code:
 ```typescript
...

 const model = new ChatAnthropic();
 const chain = new LLMChain({
    prompt: ChatPromptTemplate.fromMessages(messages),
    llm: model
 });

...
```

Packages versions:
    "langchain": "^0.1.3",
    "@langchain/anthropic": "^0.0.9",
    "@langchain/community": "^0.0.17",
    "@langchain/core": "^0.1.15",
    "@langchain/openai": "^0.0.12",

**Comment by jacoblee93** [comment]
Will have a look at this today, thanks for reporting!

**Comment by AdiletBekmuratov** [comment]
@jacoblee93 Thank you very much! If someone encounters the same issues with types, they should follow the instructions from that link. Even after that, if there is still an error, try deleting the node_modules directory (and even the *.lock file) and reinstalling the packages. These steps helped me 🙌

> Will have a look at this today, thanks for reporting!
> 
> Did you lock core to a specific version as shown here?
> 
> https://js.langchain.com/docs/get_started/installation#installing-integration-packages

**Comment by khanguslee** [comment]
To help other people updating langchain, I'm also in the process of updating langchain from v0.0.144 to 0.214 (to prep updating to 0.3 eventually), and I encountered this issue. To resolve this, I had to set the resolutions/override package versions to the corresponding package versions set out in the `langchain/package.json` file.

For example, `v0.0.204` (https://github.com/langchain-ai/langchainjs/blob/25662ed4152d0357763f046ce027ceb758dcfa57/langchain/package.json#L1177-L1179)
```json
{
  ...,
  "resolutions": {
    "@langchain/community": "0.0.2",
    "@langchain/core": "0.0.11",
    "@langchain/openai": "0.0.2",
  }
}

```

And last example, `v0.0.214` (https://github.com/langchain-ai/langchainjs/blob/9487f48b5c879e5f8a13d0606dab19462578181b/langchain/package.json#L1193-L1195)
``` json
{
  ...,
  "resolutions": {
    "@langchain/community": "0.0.13",
    "@langchain/core": "0.1.5",
    "@langchain/openai": "0.0.9",
  }
}
```

Setting the wrong package versions will throw a similar error to https://github.com/langchain-ai/langchainjs/issues/4082 which means you've set your package resolution to the incorrect value.
